scripts/pre_commit_linter.py

```diff
@@ -44,12 +44,11 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+import sys
+import time
 import argparse
 import os
 import subprocess
-import sys
-import time
```

Comment:
argparse became a standard module in Python 2.7, I believe -- and when I ran the linter on my computer, I didn't get the wrong-import-position errors. Python 2.7 is the version we assume, in general; is this what you're using?